test/internet/test-tls-canonical-ip.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+require('../common');
+
+// Test conversion of IP addresses to the format returned
+// for addresses in Subject Alternative Name section
+// of a TLS certificate
+
+const assert = require('assert');
+const tls = require('internal/tls');
```

Comment:
Does this test work without `// Flags: --expose-internals`? (See [the testing guide](https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#flags))

Reply:
You're right, it actually doesn't. I have added that flag now. While debugging this I looked at the other tests in the test/internet/ directory and realised that they all needed internet access (which my test doesn't). I therefore moved the test to test/parallel which should make sure that it gets run by `make test`